.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Question
+about: 'Please use Stack Overflow for questions #office-ui-fabric-react'
+
+---
+
+Stuff about what happens to questions here
```

Comment:
Did you mean to commit this?

Reply:
I just created a PR with all the files so we could edit them....wasn't ready for merge...sorry :D

Reply:
Gotta be careful with easy looking PRs, people will want to get them in and out of the queue ;)

Reply:
hah..yup! should have tossed the label on earlier